### ISSUE TITLE
Update values case is changed

### ISSUE THREAD

**Issue by qa7720** [IngoGege/xXMLConfigFile#17]
When a key value only changes it's case it won't be updated.
This can be solved by setting the comparison case sensitive in Test-TargetResource
`elseif ($CurrentValue -ceq $Value)`

Maybe you can fix this in one of the next versions.

Thanks

**Comment by IngoGege** [comment]
@qa7720 yeah, I also was thinking about checking the whole module as XML IS case-sensitive...
